examples/ganache/index.mjs

```diff
@@ -0,0 +1,41 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+import ethers from 'ethers';
+import ganache from 'ganache-core';
+const stdlib = loadStdlib();
+
+(async () => {
+  // OPTIONAL GANACHE CONFIG THAT CAN BE PASSED into ganache.provier()
+  const options = {
+    total_accounts: 1000,
+    gasPrice: '21000',
+    gasLimit: '5000000',
+    hardfork: "petersburg",
+    default_balance_ether: 7777777,
+    port: 7545,
+    ws: true,
+    vmErrorsonRPCResponse: true,
+    debug: true,
+    allowUnlimitedContractSize: true
+  };
+  //pass ganacheProvider into setProvider to run a reach program in a ganache env
+  const ganacheProvider = new ethers.providers.Web3Provider(ganache.provider(options));
+  await stdlib.setProvider(ganacheProvider);
+  const accAlice = await stdlib.connectAccount(ganacheProvider.getSigner(999));
```

Comment:
Can you verify that `newTestAccount` works?

Reply:
it would throw an error "Having trouble with newTestAccount 0x2930.." when i try to use it as a signer, connectAccount was my work around that error